src/components/BarChart/examples/06.all-the-things.jsx

```diff
@@ -1,19 +1,16 @@
 /* eslint-disable comma-spacing */
 
 import React from 'react';
-import {
-	BarChart,
-	chartConstants,
-} from '../../../index';
+import { BarChart, chartConstants } from '../../../index';
 
 const data = [
-	{ day: 'monday'    , apples: 2000 , oranges: 3000 } ,
-	{ day: 'tuesday'   , apples: 2000 , oranges: 5000 } ,
-	{ day: 'wednesday' , apples: 3000 , oranges: 2000 } ,
-	{ day: 'thursday'  , apples: 5000 , oranges: 6000 } ,
+	{ day: 'monday', apples: 2000, oranges: 3000 },
+	{ day: 'tuesday', apples: 2000, oranges: 5000 },
+	{ day: 'wednesday', apples: 3000, oranges: 2000 },
+	{ day: 'thursday', apples: 5000, oranges: 6000 },
 ];
-const yFormatter = (d) => `${d / 1000}k`;
-const xFormatter = (d) => d.toUpperCase().slice(0, 3);
+const yFormatter = d => `${d / 1000}k`;
```

Comment:
are we cool with dumping parens around single arguments for arrow fns? I was never a fan of those parens anyway, personally.